composer.json

```diff
@@ -65,6 +65,7 @@
         "symfony/monolog-bridge": "~5.4.0",
         "symfony/polyfill-iconv": "^1.20",
         "symfony/polyfill-mbstring": "^1.20",
+        "symfony/process": "~5.4.0",
```

Comment:
The VisitorGenerator plugin already uses the same dependency, so unless we update it in core or the plugin that shouldn't cause trouble.
But we should better check if we can change the VisitorGenerator to use CLIMulti if possible to be able to remove the dependency from there. Or we could expose the Symfony Process by providing a wrapper class, so the plugin could use that one.